tests/testthat/_snaps/keep.md

```diff
@@ -3,11 +3,15 @@
     Code
       keep(1:3, ~NA)
     Condition
-      Error in `keep()`:
+      Error in `map_lgl()`:
```

Comment:
The nesting is a bit weird here, but it is good to know where the predicate failed.

Reply:
Yeah, and it was easy to implement once I refactored the code.